Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -78,8 +78,43 @@ def formula_version(formula, spec, contents = nil)
     end
   end
 
+  def fetch_pull_requests(formula)
+    GitHub.issues_for_formula(formula.name, tap: formula.tap).select do |pr|
+      pr["html_url"].include?("/pull/")
```

Comment:
If you really cared you could do this through another parameter to restrict the search to just PRs.